docs/Contributing.md

```diff
@@ -0,0 +1,63 @@
+# Contributing
+
+A big welcome and thank you for considering contributing to the SOCIS website! It’s people like you that make it a reality for users in our community.
+
+Reading and following these guidelines will help us make the contribution process easy and effective for everyone involved. 
+
+It also communicates that you agree to respect the time of the developers managing and developing these open source projects. 
+
+In return, we will reciprocate that respect by addressing your issue, assessing changes, and helping you finalize your pull requests.
+
+## Quicklinks
+
+* [Code of Conduct](#code-of-conduct)
+* [Getting Started](#getting-started)
+    * [Issues](#issues)
+    * [Pull Requests](#pull-requests)
+* [Getting Help](#getting-help)
+
+## Code of Conduct
+
+We take our open source community seriously and hold ourselves and other contributors to high standards of communication. 
+By participating and contributing to this project, you agree to uphold our [Code of Conduct](https://github.com/Botomir/Botomir/blob/main/CODE_OF_CONDUCT.md).
```

Comment:
Code of conduct should be in this project.

Reply:
Added